bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -441,6 +454,32 @@ spec:
                   requests:
                     cpu: 20m
                     memory: 300Mi
+              - name: kube-rbac-proxy
+                image: quay.io/openshift/origin-kube-rbac-proxy:latest
+                imagePullPolicy: IfNotPresent
+                args:
+                  - --secure-listen-address=0.0.0.0:8443
+                  - --upstream=http://127.0.0.1:9182/
```

Comment:
Looks like the `windows-service` is now [getting tls_server_config](https://github.com/openshift/windows-machine-config-operator/pull/2156/files#diff-13dd767f866337233005dff5673462886531a69cd167f2e4e03dcfcb20f8652aR1) so that it will use it in the default port,  that is 9182. 

Hence, the upstream address in the proxy may need the same set of credentials to communicate with the local port opened by the  `windows-service`.  As per docs:

```
      --upstream-client-cert-file string            If set, the client will be used to authenticate the proxy to upstream. Requires --upstream-client-key-file to be set, too.
      --upstream-client-key-file string             The key matching the certificate from --upstream-client-cert-file. If set, requires --upstream-client-cert-file to be set, too.
 ```